account_invoice_supplier_ref_unique/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,5 @@
+* Marc Cassuto <marc.cassuto@savoirfairelinux.com>
+* Mathieu Benoit <mathieu.benoit@savoirfairelinux.com>
+* Alexis de Lattre <alexis.delattre@akretion.com>
+* Thomas Binsfeld <thomas.binsfeld@acsone.eu>
+* Andrea Stirpe <a.stirpe@onestein.nl>
```

Comment:
@miguels73 Please add you as a contributor on `CONTRIBUTORS.rst`.

Reply:
Added